src/main/java/javax/validation/Configuration.java

```diff
@@ -127,6 +129,19 @@
 	T clockProvider(ClockProvider clockProvider);
 
 	/**
+	 * Adds a value extractor. Has priority over any extractors for the same
```

Comment:
I missed it the first time but I think `any extractor` without an s would be more correct.